src/test/scala/SMTLIB2Spec.scala

```diff
@@ -209,6 +209,9 @@ class SMTLIB2Spec extends AnyFlatSpec {
   "test-multiply-divide.ucl" should "verify all but one assertions." in {
     SMTLIB2Spec.expectedFails("./test/test-multiply-divide.ucl", 1)
   }
+  "test-multiply-divide-subtract-chaining.ucl" should "verify all assertions." in {
+    SMTLIB2Spec.expectedFails("./test/test-multiply-divide-subtract-chaining.ucl", 0)
```

Comment:
```suggestion
    SMTLIB2Spec.expectedFails("./test/test-multiply-divide-subtract-chaining.ucl", 1)
```
(see suggested addition to test)